README.md

```diff
@@ -23,7 +23,7 @@ Running benchmarks with 1 thread, 10000000 rows, and 10 columns.
 ...
 Running benchmarks with 10 threads, 10000000 rows, and 10 columns.
 ```
-It returns an object with a class for which the print and plot methods have overridden definitions.
+It returns an object with print and plot methods.
```

Comment:
I thought we did not use the original definition of those methods, or at least in the case of the `print` function as even without the definition I 'provide' it would run with the default? 🤔

Reply:
you have provided a method print.Your_Class which is called instead of print.default

Reply:
(print.default has not been overridden, it is just no longer called)

Reply:
> you have provided a method print.Your_Class which is called instead of print.default

That makes sense given that there is no default method for `print.data_table_threads_benchmark` - Thanks for explaining!